src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/CountFunctions.txt

```diff
@@ -27,3 +27,27 @@ Errors: Error 22-25: Expected boolean. We expect a boolean (true/false) at this
 
 >> CountIf([1, 2, 3, 4], "abc")
 Errors: Error 22-27: Expected boolean. We expect a boolean (true/false) at this point in the formula.
+
+>> CountA([1,2,3,Blank()])
+3
+
+>> CountA([1,2,3,Blank(),Blank(),Blank()])
+3
+
+>> CountA([1,2,3,If(1<0,4),Blank()])
+3
+
+>> CountA(Sort([-2, -1, 0, 1, 2], 1 / Value))
+#Error
+
+>> CountA(LastN([1,2,3,4], Blank()))
+Blank()
+
+>> CountA([1, 2, 1/0, 4, 5])
+5
```

Comment:
Can you double check this one in Power apps (with error handling feature enabled - see comment at top of #284). 

If any element is an error, it should return error.

Reply:
Thanks for clarifying. In PApps both Count and CountA will display an error message. This has been taken into account.